Pokedex/Storyboard/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,242 @@
+                                    </constraints>
+                                </tableViewCellContentView>
+                                <connections>
+                                    <outlet property="nameLabel" destination="Eez-9c-oi6" id="VmL-7E-Ab4"/>
+                                    <outlet property="pokemonSprite" destination="zon-c1-52H" id="yQG-wy-Ney"/>
+                                    <segue destination="K2I-Fx-IFg" kind="show" identifier="DetailViewSegue" id="b0O-hf-zi9"/>
+                                </connections>
+                            </tableViewCell>
+                        </prototypes>
+                        <connections>
+                            <outlet property="dataSource" destination="gWc-vE-QI4" id="i0N-4l-YEx"/>
+                            <outlet property="delegate" destination="gWc-vE-QI4" id="sVD-pH-cBf"/>
+                        </connections>
+                    </tableView>
+                    <navigationItem key="navigationItem" title="Pokedex" id="TD3-dI-LNZ">
+                        <barButtonItem key="rightBarButtonItem" systemItem="search" id="gZI-uO-ah2">
+                            <connections>
+                                <segue destination="VxI-jV-34x" kind="show" identifier="SearchSegue" id="6OL-En-bLd"/>
+                            </connections>
+                        </barButtonItem>
+                    </navigationItem>
+                </tableViewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="k4r-Ho-KDh" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="952.79999999999995" y="187.55622188905548"/>
+        </scene>
+        <!--Pokemon Search-->
+        <scene sceneID="pWa-1r-EG5">
+            <objects>
+                <viewController id="VxI-jV-34x" customClass="PokemonSearchViewController" customModule="Pokedex" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="UTS-Ex-pdT">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <searchBar contentMode="redraw" translatesAutoresizingMaskIntoConstraints="NO" id="dzC-6Y-48b">
+                                <rect key="frame" x="0.0" y="118" width="375" height="44"/>
+                                <textInputTraits key="textInputTraits"/>
+                            </searchBar>
+                            <stackView opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" axis="vertical" alignment="top" spacing="42" translatesAutoresizingMaskIntoConstraints="NO" id="dce-cS-RD8">
```

Comment:
You need to work on Auto Layout. For a better results stackview's alignment & distribution & spacing need modification.